[14.0][ADD] mrp_subcontracting_resupply_link: New module. (#837)
OCA/manufacture: mrp_subcontracting_resupply_link/readme/USAGE.rst

#. Go to *Manufacturing > Products > Products* and create a new subcontracted product with some Vendor (Purchase tab).
#. Go to *Manufacturing > Products > Products* and create a new one with "Resupply Subcontractor on Order" route checked. [Component]
#. Go to *Manufacturing > Products > Bill of materials* and create a new one with type = "Subcontracting" and set the subcontractor and component previously created.
#. Go to *Purchase > Orders > Requests for Quotation* and create a new one to the subcontractor and add subcontracted product.
#. Click on "Confirm order" button.

> #. A message linked to resupply(s) will have been added to purchase order.

Reviewer: ```suggestion
#. A message with the link to the resupply(ies) is added in the purchase order.
```

> #. A "Resupply(s)" smart-button will also be displayed in the purchase order.

Reviewer: ```suggestion
#. A "Resupply(ies)" smart-button is also displayed in the purchase order.
#. Clicking on it, it navigates to the linked pickings.
```

> #. A message linked to supplying raw material will have been added in the resupply.

Reviewer: ```suggestion
#. A message with the link to the PO is added in the resupply picking.
```
